packages/core/src/App/Containers/AccountSignupModal/account-signup-modal.jsx

```diff
@@ -80,6 +90,22 @@ const AccountSignup = ({ enableApp, isModalVisible, clients_country, onSignup, r
         });
     }, []); // eslint-disable-line react-hooks/exhaustive-deps
 
+    React.useEffect(() => {
```

Comment:
We don’t need this effect, this code be inside `onResidenceSelection`

Reply:
Couldn't completely resolve this. I am able to push the value in the selected_residence usestate, but not able to retrieve the same value in the same function execution instance.

Reply:
Nevermind about my previous comment, I managed to resolve this.